ansible/roles/screenly/tasks/main.yml

```diff
@@ -9,6 +9,14 @@
     - .config
     - screenly_assets
 
+- name: Copy Screenly default config
```

Comment:
We can remove this block entirely. It's done [here](https://github.com/Screenly/screenly-ose/blob/experimental/bin/start_server.sh#L8-L10).